### ISSUE TITLE
Syntax error on import 

### ISSUE THREAD

**Issue by AndrewKralovec** [kekeh/mydatepicker#160]
I'm getting a syntax error when importing the module into my project . Any ideas on the cause ? 

**Console**   
`SyntaxError: export declarations may only appear at top level of a module`

**App.module.ts** 
```
......
import { MyDatePickerModule } from 'mydatepicker';

@NgModule({
    bootstrap: [ AppComponent ],
    declarations: [
        HomeComponent
        .........
    ],
    imports: [
        UniversalModule, 
        MyDatePickerModule, 

```

**Webpack config file** 

```
var isDevBuild = process.argv.indexOf('--env.prod') < 0;
var path = require('path');
var webpack = require('webpack');
var nodeExternals = require('webpack-node-externals');
var merge = require('webpack-merge');
var allFilenamesExceptJavaScript = /\.(?!js(\?|$))([^.]+(\?|$))/;

// Configuration in common to both client-side and server-side bundles
var sharedConfig = {
    context: __dirname,
    resolve: { extensions: [ '', '.js', '.ts' ] },
    output: {
        filename: '[name].js',
        publicPath: '/dist/' // Webpack dev middleware, if enabled, handles requests for this URL prefix
    },
    module: {
        loaders: [
            { test: /\.ts$/, include: /ClientApp/, loaders: ['ts?silent=true', 'angular2-template'] },
            { test: /\.html$/, loader: 'html' },
            { test: /\.css$/, loader: 'to-string!css' },
            { test: /\.(png|jpg|jpeg|gif|svg)$/, loader: 'url', query: { limit: 25000 } }
        ]
    }
};
```

**Comment by kekeh** [comment]
Can you check this: https://github.com/kekeh/mydatepicker/tree/gh-pages

Webpack config file is: config/webpack.build.js

This working

**Comment by kekeh** [comment]
I published new version 1.4.0 which restructure npm package. You can also exclude node_modules folder in the tsconfig.json file.

I will close this issue. If you have still problems you can reopen the issue.

**Comment by fidelsam1992** [comment]
Please re-open this issue.

#### I have got the same error : 
```SyntaxError: export declarations may only appear at top level of a module```

#### tsconfig.json
```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "baseUrl": ".",
    "declaration": false,
    "emitDecoratorMetadata": true,
    "importHelpers": true,
    "experimentalDecorators": true,
    "lib": [
      "dom",
      "es2015"
    ],
    "module": "commonjs",
    "moduleResolution": "node",
    "sourceMap": true,
    "target": "es5",
    "skipLibCheck": true,
    "stripInternal": true,
    "noImplicitAny": false,
    "types": [
      "@types/meteor",
      "@types/mocha",
      "@types/chai",
      "@types/sinon",
      "@types/meteor-publish-composite"
    ]
  },
  "include": [
    "imports/**/*.ts",
    "client/**/*.ts",
    "server/**/*.ts"
  ],
  "exclude": [
    "node_modules"
  ],
  "compileOnSave": false,
  "atom": {
    "rewriteTsconfig": false
  }
}
```
